docs/content/templates/functions.md

```diff
@@ -758,6 +758,52 @@ This can be useful if you want to use Gravatar for generating a unique avatar:
 <!-- returns the string "c8b5b0e33d408246e30f53e32b8f7627a7a649d4" -->
 ```
 
+## Reflection
+
+### typeOf
+
+Takes an interface and returns a string representation of the type. For pointers, this will return a type prefixed with an asterisk(*). So a pointer to type Foo will be *Foo.
```

Comment:
Use backticks to keep the markdown from thinking you mean to use italics around the asterisks.  For example,

```
with an asterisk(`*`).  So a pointer to a type `Foo` will be `*Foo`.
```